.github/workflows/pr-title-check.yml

```
@@ -0,0 +1,22 @@
+  pull_request:
+    types: [opened, edited, reopened, synchronize]
+
+jobs:
+  check-title:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check PR Title Format
+        uses: amannn/action-semantic-pull-request@v5
+        with:
+          types: | 
+          docs 
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 15: could not find expected ':'

(syntax-check)